interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -735,6 +735,49 @@
                   </constraint>
                 </properties>
               </leafNode>
+              <node name="options">
+                <properties>
+                  <help>DNS server options</help>
+                </properties>
+                <children>
+                  <leafNode name="ecs-add-for">
+                    <properties>
+                      <help>List of client netmasks for which EDNS Client Subnet will be added</help>
```

Comment:
Maybe just "EDNS client subnet"?

Even though it's a multi-node, we generally don't say "list" or similar because for many such options it's common to have just one.

Reply:
fixed